gcloud-java-examples/src/main/java/com/google/cloud/examples/datastore/DatastoreExample.java

```diff
@@ -131,25 +158,37 @@ public void run(Transaction tx, Key userKey, String... args) {
     }
 
     @Override
-    public String getRequiredParams() {
-      return "";
+    Void parse(String... args) throws Exception {
+      return null;
     }
   }
 
-  private static class AddAction implements DatastoreAction {
+  /**
+   * This class adds a comment for a user. If the user does not exist its entity is created.
+   */
+  private static class AddCommentAction extends DatastoreAction<String> {
     @Override
-    public void run(Transaction tx, Key userKey, String... args) {
+    public void run(Transaction tx, Key userKey, String content) {
       Entity user = tx.get(userKey);
       if (user == null) {
         System.out.println("Adding a new user.");
-        user = Entity.builder(userKey)
-            .set("count", 1L)
-            .build();
+        user = Entity.builder(userKey).set("count", 1L).build();
```

Comment:
Nothing, 1 is automatically promoted to long. `1L` was used to show that the method takes long parameter.